p4-16/spec/grammar.mdk

```diff
@@ -328,12 +331,15 @@ typeParameterList
 realTypeArg
     : DONTCARE
     | typeRef
+    | VOID
     ;
 
 typeArg
     : DONTCARE
     | typeRef
     | nonTypeName
+        // This is necessary because template arguments may introduce the return type
```

Comment:
I believe that the comment applies to the previous line

Reply:
I just removed the comment.  I only left it in in case it helped explain the reason for the VOID case of typeArg on the next line.